modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.tsx

```diff
@@ -315,10 +315,10 @@ const Main = ({
 				type="hidden"
 				value={
 					multiple
-						? newValue
+						? JSON.stringify(newValue)
 						: newValue?.[0] === 'chooseAnOption'
 						? undefined
-						: newValue
+						: JSON.stringify(newValue)
```

Comment:
@rebsilva could you please review or ask someone to review this change?

Reply:
sure! i will take a look :cherry_blossom:

Reply:
seems fine to me! just make sure to run the forms and data-engine suites to make sure everything is working as intended after this change, thanks!